pghoard/monitoring/prometheus.py

```diff
@@ -39,7 +39,9 @@ def _update(self, metric, value, tags):
         ts = str(int(time.time())) + "000"
 
         metric = metric.replace(".", "_")
-        tags = {**self._tags, **tags}
+        metric = metric.replace("-", "_")
```

Comment:
Duplicated line

Reply:
actually its not a duplicate line, we need to replace both periods and hyphens as neither are supported for metric name